README.md

```diff
@@ -7,18 +7,37 @@ Include `layer:nginx` in your `layer.yaml`
 
 ## adding Vhosts
 
-To proxy requests through NGINX to another application create a  
+To proxy requests through NGINX to another application create a
 `site.toml` file with the following:
 
 ```toml
 "server_name" = "mybouncer.example.com"
-"packages" = ["php5-fpm", "php5-mysql"]
 "app_path" = "/srv/myapp"
 ```
 
 The variables in `site.toml` will be available when configuring the NGINX
 `templates/vhost.conf`.
 
+If additional packages are required (ie php-mysql, php-mcrypt) in your `layer.yaml` add:
```

Comment:
Actually I would probably invert this. NGINX is the one known package we expect to come with this layer, so move that to the layer.yaml install bits, and make the extra packages part of charm config, and use the apt layer to install the extra packages.